ui/app/src/components/WorkflowCancellationDialog/utils.ts

```diff
@@ -25,7 +24,7 @@ export type Source = SandboxItem[];
 export type Return = Omit<Source, 'error'>;
 
 export interface WorkflowCancellationDialogUIProps {
-  resource: Resource<Return>;
+  items: Return;
```

Comment:
```suggestion
  items: SandboxItem[];
```

Is this `Return` used elsewhere? Remove if possible.